packages/subgraph/src/dao/dao_v1.ts

```diff
@@ -11,7 +11,7 @@ import {
   SignatureValidatorSet,
   StandardCallbackRegistered,
   CallbackReceived
-} from '../../generated/templates/DaoTemplate/DAO';
+} from '../../generated/templates/DaoTemplateV1/DAO';
```

Comment:
Is the versioning number following the @aragon/osx npm number?
Should we use 
V1_0 and V1_2  or 
V1_0_0 and V1_2_0 ?

Reply:
good point. Done